castle/cms/interfaces/controlpanel.py

```diff
@@ -143,6 +143,27 @@ class ISecuritySchema(controlpanel.ISecuritySchema):
         )
     )
 
+    pwexpiry_enabled = schema.Bool(
+        title=u'Enable Password Expiration',
+        description=u'Toggle whether or not password expiration \
+        and password history are enabled.',
+        default=False
+    )
+
+    pwexpiry_validity_period = schema.Int(
+        title=u'Password Validity Period',
+        description=u'Number of days for password validity \
+        (set to 0 to disable).',
```

Comment:
Original functionality. But if for some reason someone wanted to prevent users from reusing passwords but didn't want passwords to expire, they could do that.

Reply:
As long as that's in the docs ... :)